project/accounts/templates/accounts/settings.html

```diff
@@ -1,81 +1,98 @@
-    <div class="white-background-input-field input-field col s6">
-        <input class="save-account" id="last_name" type="text" value="{{this.model.get('last_name')}}">
-        <label for="last-name">{% trans "Last Name" %}</label>
-    </div>
-{% endverbatim %}
-</script>
-
-<script id="settings-base" type="text/template">
-{% verbatim %}
-    <div class="subtitle-lato">{% trans "Personal" %} </div>
-        <div class="white-background-input-field input-field col s12">
-            <input id="email" type="text" class="validate" value="{{this.model.get('email')}}" disabled>
-            <label for="email">{% trans "Email" %}</label>
-        </div>
-        <div class="white-background-input-field input-field col s12">
-            <input id="username" type="text" class="validate" value="{{this.model.get('username')}}" disabled>
-            <label for="username">{% trans "Username" %} </label>
-        </div>
-        <div id="settings-1"></div>
-    </div>
-{% endverbatim %}
-</script>
-
-<script id="settings-template" type="text/template">
-    <div class="container row">
-        <div class="section row"></div>
-        <div class="z-depth-1 white">
-            <div class="container">
-                <div class="section center">
-                    <div class="title-lato">{% trans "My Settings" %} </div>
-
+{% block content %}
+<section style='display: flex;justify-content: center; padding-top: 20px;'>
+    <div class="setting card" style="width: 75%;">
+        <div class="card-content">
+            <div class="step-title">
+                <div class="section">
+                    <div class="title-lato center">
+                        Account Settings
+                    </div>
                 </div>
-                <div class="row">
-                    <div id="settings-el"></div>
+                <div class="accent-line"></div>
+                {{form.non_field_errors}}
+                <form method="POST" enctype="multipart/form-data">
+                    {% csrf_token %}
+                    <div class='row'>
+                        <div class=" col s12 m6 ">
+                            <span class='error'>{{form.first_name.errors}}</span>
+                            {{form.first_name.label_tag}}
+                            {{form.first_name}}
+                        </div>
+                        <div class=" col s12 m6 ">
+                            <span class='error'>{{form.last_name.errors}}</span>
+                            {{form.last_name.label_tag}}
+                            {{form.last_name}}
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class=" col s12 m6 ">
+                            <span class='error'>{{form.profile_image.errors}}</span>
+                            {{form.profile_image.label_tag}}
+                            {{form.profile_image}}
+                        </div>
+                    </div>
```

Comment:
When I press the `Choose File` button, the file manager pop-up is not open.

Is it the same for you? Could you test uploading the profile image?

<img width="227" alt="image" src="https://user-images.githubusercontent.com/16780927/197038025-0600310a-7923-4d8f-a3a0-5712f5e96d5d.png">

Reply:
The file manager opens for me in Chromium. What browser are you using? Note that we are using the standard browser file chooser, so no custom logic is in our code.

Reply:
I updated my browser and it was fixed.